### PR TITLE
Feat/update alias output examples

### LINKED ISSUE #127: Add Create an Alias Output example
## Description

- [x] Add example how to create an Alias output.
- [x] Place alias output examples in Outputs and Transactions.
~- Remove creation of alias output from Mint Native Tokens example and add as assumption.~ Needs more discussion

## Motivation

Currently Mint Native Tokens is not just about native tokens, but also about alias outputs. Also the Destroy an Alias Output cognitively kind of stands by itself right now, it would be more clear with an associated Create an Alias Output.

## Are you planning to do it yourself in a pull request?

Yes.

### PR DESCRIPTION
**Pull request by jlvandenhout**
# Description of change

- Created examples how to create alias outputs.
- Added examples how to destroy alias outputs.
- Moved alias output examples to Outputs and Transactions
- Aligned output examples between languages, except Rust. Full alignment between examples will be handled in separate PRs.

## Links to any relevant issues

fixes iotaledger/iota-sdk#127

## Type of change

Choose a type of change, and delete any options that are not relevant.
- Documentation Fix

## How the change has been tested

- Local runs of the code listings.
- Local Wiki build to check correct display of code listings and example output.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: wallet/documentation/docs/libraries/rust/how_to/outputs_and_transactions/_06_create_alias/_code_sample.mdx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+<DotEnvWarning/>
+
+<CodeBlock className="language-rust">
+    {CodeSample}
+</CodeBlock>
+
+### Run the Example
+
+Before you run the example you should update the `alias_id` to one available in your account.
```
Comment (Thoralf-M) [inline wallet/documentation/docs/libraries/rust/how_to/outputs_and_transactions/_06_create_alias/_code_sample.mdx]: Copy paste error? There is nothing with alias_id in this example

File: wallet/documentation/docs/how_tos/outputs_and_transactions/06_create_alias_output.mdx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import NodejsCode from "../../libraries/nodejs/how_to/outputs_and_transactions/_06_create_alias/_code_sample.mdx";
+import NodejsOutput from "../../libraries/nodejs/how_to/outputs_and_transactions/_06_create_alias/_expected_output.md";
+import PythonCode from "../../libraries/python/how_to/outputs_and_transactions/_06_create_alias/_code_sample.mdx";
+import PythonOutput from "../../libraries/python/how_to/outputs_and_transactions/_06_create_alias/_expected_output.md";
+import JavaCode from "../../_admonitions/_guide_coming_soon.md";
+import JavaOutput from "../../_admonitions/_guide_coming_soon.md";
+import RustCode from "../../libraries/rust/how_to/outputs_and_transactions/_06_create_alias/_code_sample.mdx";
+import RustOutput from "../../libraries/rust/how_to/outputs_and_transactions/_06_create_alias/_expected_output.md";
+
+You can create an alias output using the `Account.create_alias(alias_id, options)` function.
```
Comment (Thoralf-M) [inline wallet/documentation/docs/how_tos/outputs_and_transactions/06_create_alias_output.mdx]: Such a function doesn't exist
There is `create_alias_output()`, but it also doesn't accept an alias_id

Comment (jlvandenhout) [reply wallet/documentation/docs/how_tos/outputs_and_transactions/06_create_alias_output.mdx]:
```suggestion
You can create an alias output using the `Account.create_alias_output(alias_options, transaction_options)` function.
```

File: wallet/documentation/docs/libraries/rust/how_to/outputs_and_transactions/_06_create_alias/_code_sample.mdx (hunk 1 above)
Comment (jlvandenhout) [reply wallet/documentation/docs/libraries/rust/how_to/outputs_and_transactions/_06_create_alias/_code_sample.mdx]:
```suggestion
```

File: sdk/src/wallet/bindings/nodejs/examples/destroy-alias.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+/**
+ * This example will destroy an alias
+ */
+const getUnlockedManager = require('./account-manager');
+
+async function run() {
+    // Replace with an existing alias ID in your account
+    const ALIAS_ID = '0x08e6210d29881310db2afde095e594f6f006fcdbd06e7a83b74bd2bdf3b5190d0e0200000000'
```
Comment (Thoralf-M) [inline sdk/src/wallet/bindings/nodejs/examples/destroy-alias.js]: This is no alias id

File: sdk/src/wallet/bindings/python/examples/destroy_alias.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+from iota_wallet import IotaWallet
+
+# In this example we will destroy an alias output
+
+# TODO: replace with your own values.
+ALIAS_ID = "0x08429fe5864378ce70699fc2d22bb144cb86a3c4833d136e3b95c5dadfd6ba0cef0500000000"
```
Comment (Thoralf-M) [inline sdk/src/wallet/bindings/python/examples/destroy_alias.py]: Same here, that's not an alias id

File: sdk/src/wallet/bindings/nodejs/examples/destroy-alias.js (hunk 3 above)
Comment (jlvandenhout) [inline sdk/src/wallet/bindings/nodejs/examples/destroy-alias.js:8]:
```suggestion
    const ALIAS_ID = '0x982667c59ade8ab8a99188f4de38c68b97fc2ca7ba28a1e9d8d683996247e152'
```

File: sdk/src/wallet/bindings/python/examples/destroy_alias.py (hunk 4 above)
Comment (jlvandenhout) [reply sdk/src/wallet/bindings/python/examples/destroy_alias.py]:
```suggestion
ALIAS_ID = "0x982667c59ade8ab8a99188f4de38c68b97fc2ca7ba28a1e9d8d683996247e152"
```

File: sdk/src/wallet/bindings/nodejs/examples/destroy-alias.js (hunk 3 above)
Comment (jlvandenhout) [reply sdk/src/wallet/bindings/nodejs/examples/destroy-alias.js]:
```suggestion
    const ALIAS_ID = '0x982667c59ade8ab8a99188f4de38c68b97fc2ca7ba28a1e9d8d683996247e152'
```

**Review by Thoralf-M — DISMISSED** [review]

File: sdk/examples/wallet/create_alias.rs [hunk 5]
```diff
@@ -0,0 +1,41 @@
+// Copyright 2022 IOTA Stiftung
```
Comment (thibault-martinez) [inline sdk/examples/wallet/create_alias.rs]:
```suggestion
// Copyright 2023 IOTA Stiftung
```

**Review by thibault-martinez — APPROVED** [review]

**Review by Thoralf-M — DISMISSED** [review]

**Review by thibault-martinez — CHANGES_REQUESTED** [review]
```
error: couldn't read sdk/examples/create_alias.rs: No such file or directory (os error 2)
```

**Review by DaughterOfMars — DISMISSED** [review]

File: sdk/Cargo.toml [hunk 6]
```diff
@@ -456,6 +456,11 @@ name = "destroy_foundry"
 path = "examples/wallet/15_destroy_foundry.rs"
 required-features = [ "stronghold" ]
 
+[[example]]
+name = "create_alias"
+path = "examples/wallet/create_alias.rs"
```
Comment (kwek20) [inline sdk/Cargo.toml:525]: do we want to assign a number?

Comment (thibault-martinez) [reply sdk/Cargo.toml:525]: We're going to reorganise the examples that are used by the wiki soon, we may completely drop the numbers somehow so I wouldn't necessarily do it.

**Review by thibault-martinez — CHANGES_REQUESTED** [review]
Conflicts

**Review by kwek20 — DISMISSED** [review]

**Review by Thoralf-M — CHANGES_REQUESTED** [review]
AccountManager got renamed to Wallet in the Rust crate, needs to be changed

**Review by Thoralf-M — DISMISSED** [review]

File: sdk/examples/wallet/create_alias.rs [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+// Rename `.env.example` to `.env` first
+
+use std::env;
+
+use dotenvy::dotenv;
+use iota_sdk::wallet::{Result, Wallet};
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    // This example uses dotenv, which is not safe for use in production
```
Comment (DaughterOfMars) [inline sdk/examples/wallet/create_alias.rs]: Needs to be rebased and updated.

**Review by DaughterOfMars — DISMISSED** [review]

File: sdk/examples/wallet/create_alias.rs (hunk 7 above)
Comment (thibault-martinez) [reply sdk/examples/wallet/create_alias.rs]: @jlvandenhout in cased you missed it

Comment (jlvandenhout) [reply sdk/examples/wallet/create_alias.rs]: I rebased shortly after the comment and got no conflicts 🤔 So if @Alexandcoats is ok with it we can markt his resolved.

Comment (thibault-martinez) [reply sdk/examples/wallet/create_alias.rs]: Ah no, Alex meant that we changed this comment in all other examples so it would be nice to adapt it

**Review by kwek20 — DISMISSED** [review]

File: sdk/examples/wallet/create_alias.rs [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+    dotenv().ok();
```
Comment (jlvandenhout) [inline sdk/examples/wallet/create_alias.rs]:
```suggestion

use iota_sdk::wallet::{Result, Wallet};

#[tokio::main]
async fn main() -> Result<()> {
    // This example uses secrets in environment variables for simplicity which should not be done in production.
    dotenvy::dotenv().ok();
```

File: sdk/examples/wallet/create_alias.rs (hunk 7 above)
Comment (jlvandenhout) [reply sdk/examples/wallet/create_alias.rs]: Ah got it. Changed it.

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**